seatunnel-connectors-v2/connector-file/connector-file-base/pom.xml

```diff
@@ -93,5 +93,16 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi</artifactId>
+            <scope>compile</scope>
```

Comment:
You can delete this, the scope should be managed by the `dependency manager` in the root directory, and the default `compiler`